lib/exrm/plugins/conform.ex

```diff
@@ -34,6 +34,24 @@ defmodule ReleaseManager.Plugin.Conform do
         warn "Conform: No schema found, conform will not be packaged in this release!"
         false
       true ->
+        {:ok, curr_path} = File.cwd
+        imports = Keyword.get(merged, :import, [])
+        File.cd("_build/" <> (Mix.env |> Atom.to_string) <> "/lib")
+
+        build_files = Enum.reduce(imports, [], fn(deps_app, acc) ->
+          {:ok, files_list} = :file.list_dir("#{deps_app}/ebin")
+          files_list = Enum.map(files_list, fn(filename) -> ("#{deps_app}/ebin/" <> "#{filename}") |> to_char_list end)
+          :lists.append(acc, files_list)
+        end)
+
+        case build_files do
+          [] ->
+            File.cd(curr_path)
+          _ ->
+            :zip.create("#{curr_path}/rel/#{app}/releases/#{version}/app.schema.deps.ez" |> to_char_list, build_files)
```

Comment:
I think, it would be better to create zip archive in rel/files (as schema) and add it to overlays, as schema.